generator/standalone.cc

```diff
@@ -1,9 +1,8 @@
-#include <fstream>
-#include <standalone.h>
-
 #include <google/protobuf/compiler/command_line_interface.h>
 #include <google/protobuf/stubs/io_win32.h>
+#include <fstream>
 #include <gapic_generator.h>
```

Comment:
Is this part of your project? If so, you probably want quotes?

Reply:
Aaaah. I'll fix that right away.